Latest version (0.9.1) results in many compile errors

I updated a .Net 8 project that was using 0.8.1 to the latest 0.9.1, and when the project is compiled, there are a ton of errors of the kind:

` error CS0103: The name 'Slices' does not exist in the current context`

Sample line where the error is generated:

`var slice = Slices.Rx.ExportPrescriptionPDF.Create(prescriptionDetail);`

I am also seeing several warnings about analyzers and source generators:

```
CSC : warning CS9057: The analyzer assembly 'e:\.nuget\packages\razorslices\0.9.1\analyzers\dotnet\cs\RazorSlices.SourceGenerator.dll' references version '4.12.0.0' of the 
 compiler, which is newer than the currently running version '4.11.0.0'.
```

```
CSC : warning RSG002: TargetPath not specified for additional file: E:\source\repos\MyApp.DocGen\DocGen.Services\bin\Debug\net8.0\Slices\Rx\ExportPrescription PDF.cshtml.
```

The RSG002 looks to be every .cshtml file that is being used as a slice.

Backing down to 0.8.1 clears the errors.

Looks like the SourceGenerator isn't working? Also, this is using VSCode (with the C# Sdk). Opening the project using VS2022 did return the same errors, but after several clean/rebuild cycles, it seems to fix the problem. (Yes, I've done a `dotnet clean` `dotnet build` several times.

Thanks

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -5,7 +5,7 @@
   <ItemGroup>
     <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.11.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Common" Version="4.8.0" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.12.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.11.0" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
     <PackageVersion Include="RazorSlices" Version="0.7.0" />

